build/upgrade_pdfjs.sh

```diff
@@ -30,12 +30,12 @@ git clone https://github.com/mozilla/pdfjs-dist.git --depth 1 --single-branch --
 echo "-----------------------------------------------------------------------------------"
 echo "Copying relevant files to third-party directory..."
 echo "-----------------------------------------------------------------------------------"
-\cp -rf pdfjs-dist/es5/build/pdf.js ${DOC_STATIC_ASSETS_PATH}
-\cp -rf pdfjs-dist/es5/build/pdf.min.js ${DOC_STATIC_ASSETS_PATH}
-\cp -rf pdfjs-dist/es5/build/pdf.worker.js ${DOC_STATIC_ASSETS_PATH}
-\cp -rf pdfjs-dist/es5/build/pdf.worker.min.js ${DOC_STATIC_ASSETS_PATH}
-\cp -rf pdfjs-dist/es5/web/pdf_viewer.css ${DOC_STATIC_ASSETS_PATH}
-\cp -rf pdfjs-dist/es5/web/pdf_viewer.js ${DOC_STATIC_ASSETS_PATH}
+\cp -rf pdfjs-dist/legacy/build/pdf.js ${DOC_STATIC_ASSETS_PATH}
```

Comment:
Can we use the non-legacy version now that we're targeting modern browsers?

Reply:
Yeah, we'd need to re-evaluate the configuration we're using for `closure-compiler` or switch to Babel to handle the minifying step.

Reply:
Ahh I see...yeah, I tried an updated jar file for the closure-compiler, but that didn't work :P So I think we'd have to switch to Babel..

Reply:
@jstoffan Is that a pretty small change that I could include in this PR?? Or should we do it in another ticket??

Reply:
That said, it can be in a follow-up PR, but that should happen prior to the release of this change if we're going to do it.